### PR TITLE
Taplytics Update: Added delayLoad and shakeMenu options and updated to latest version.

### PR DESCRIPTION
**Pull request by nemo**
Hey folks,

We've upgraded our SDK and added some options for starting up the SDK that a few clients were asking for.

The new options are:

- `delayLoad` (Number) – seconds to keep showing the splash screen until the experiments are loaded. Used for when running an experiment on the first page of the app.
- `shakeMenu` (Boolean) – If it should permanently disable the Shake Menu or not.

Thanks in advance! Let me know if this all looks good.

cc: @BayPhillips + @jonathannorris.

### CODE REVIEW

**Comment by platedbay** [comment]
+1 for this!

File: Analytics/Integrations/Taplytics/SEGTaplyticsIntegration.m [hunk 1]
```diff
@@ -94,6 +102,23 @@ - (NSString *)apiKey {
...
+    NSString *value = self.settings[@"delayLoad"];
+    
+    if (value != nil)
+        return [NSNumber numberWithInt:[value intValue]];
+    
+    return nil;
+}
+
+- (NSNumber *)shakeMenu {
+    NSString *value = self.settings[@"shakeMenu"];
```
Comment (f2prateek) [inline Analytics/Integrations/Taplytics/SEGTaplyticsIntegration.m]: I think the default serialization parses booleans as numbers so you'll have to do:

```obj-c
if ([(NSNumber *)[self.settings objectForKey:@"shakeMenu"] boolValue]) {
    return YES;
}
return NO;
```

File: Analytics/Integrations/Taplytics/SEGTaplyticsIntegration.m [hunk 2]
```diff
@@ -28,9 +28,17 @@ - (id)init {
 }
 
 - (void)start {
-  [Taplytics startTaplyticsAPIKey:[self apiKey]];
+  NSDictionary *options = [[NSMutableDictionary alloc] init];
   
-  SEGLog(@"TaplyticsIntegration initialized with api key %@", [self apiKey]);
+  if ([self delayLoad])
+      [options setValue:[self delayLoad] forKey:@"delayLoad"];
```
Comment (f2prateek) [inline Analytics/Integrations/Taplytics/SEGTaplyticsIntegration.m]: Given that the Taplytics SDK is initialized asynchronously by the Segment SDK - does this setting actually work as mentioned when being used via Segment?

Comment (nemo) [inline Analytics/Integrations/Taplytics/SEGTaplyticsIntegration.m]: We believe so. Some initial tests showed that it worked internally. As long as the users end up initialize Segment in the `didFinishLaunching:` method.

We'll be testing this with a few clients over time and will modify as we go if there's any issues.

File: Analytics/Integrations/Taplytics/SEGTaplyticsIntegration.m (hunk 1 above)
Comment (nemo) [inline Analytics/Integrations/Taplytics/SEGTaplyticsIntegration.m]: Oh cool, I'll edit that up right away.

File: Analytics/Integrations/Taplytics/SEGTaplyticsIntegration.m (hunk 2 above)
Comment (f2prateek) [inline Analytics/Integrations/Taplytics/SEGTaplyticsIntegration.m]: Just to confirm it ok to to say `[options setValue:@0 forKey:@"delayLoad"];`?

Comment (nemo) [inline Analytics/Integrations/Taplytics/SEGTaplyticsIntegration.m]: Yup!

**Comment by f2prateek** [comment]
Closed by #325

**Pull request closed without merging**